pkg/kubelet/container/helpers.go

```diff
@@ -44,6 +44,7 @@ type RuntimeHelper interface {
 	GetClusterDNS(pod *api.Pod) (dnsServers []string, dnsSearches []string, err error)
 	GetPodDir(podUID types.UID) string
 	GeneratePodHostNameAndDomain(pod *api.Pod) (hostname string, hostDomain string, err error)
+	GetExtraSupplementalGroupsForPod(pod *api.Pod) []int64
```

Comment:
I don't think this needs to be in the runtime helper interface -- why not just a utility function?